docs/framework/wcf/feature-details/using-contracts-in-workflow.md

```diff
@@ -25,17 +25,17 @@ When implementing a service, you define a number of contracts that describe the
   
 -   <xref:System.ServiceModel.Activities.Receive.ServiceContractName%2A>  
   
--   <!--zz <xref:System.ServiceModel.Activities.Receive.OperationContractName%2A>  --> `System.ServiceModel.Activities.Receive.OperationContractName`
+-   <xref:System.ServiceModel.Activities.Receive.OperationContractName%2A>
   
 -   <xref:System.ServiceModel.Activities.Receive.Action%2A>  
   
--   <!--zz <xref:System.ServiceModel.Activities.Receive.ValueType%2A> --> `System.ServiceModel.Activities.Receive.ValueType`
+-   xref:System.ServiceModel.Activities.Receive.ValueType%2A>
   
  <xref:System.ServiceModel.Activities.SendReply> Activity  
   
 -   <xref:System.ServiceModel.Activities.SendReply.Action%2A>  
   
--   <!--zz <xref:System.ServiceModel.Activities.SendReply.ValueType%2A>-->  `System.ServiceModel.Activities.SendReply.ValueType`
+-   xref:System.ServiceModel.Activities.SendReply.ValueType%2A>
```

Comment:
not sure what would be the replacement here

Reply:
I'm assuming it refers to the SendReply.ValueType property found [here](https://msdn.microsoft.com/en-us/library/system.servicemodel.activities.SendReply.valuetype).

Reply:
Same here.